src/React.AspNet/project.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "2.2.2-*",
+    "version": "2.2.2-dev-20160307-1719",
```

Comment:
Please revert these changes. `dev-build.bat` should automatically revert them back after a successful build.